halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/deploy/DeploymentEnvironmentCommand.java

```diff
@@ -35,6 +37,7 @@
 
   public DeploymentEnvironmentCommand() {
     registerSubcommand(new EditDeploymentEnvironmentCommand());
+    registerSubcommand(new HaServiceCommand());
```

Comment:
This makes the command structure `hal deploy edit ha <>`. Is this what we want the structure to be?

Reply:
I meant for the command structure to be ```hal deploy config ha <>```